docs/guide/core-targets.md

```diff
@@ -89,6 +89,11 @@ Now we have the following options:
 
 ## Script with arguments
 
+If you don't do `Target.initEnvironment()` at all, then FAKE will only use environment variables already set
+(e.g. Windows: `set mything=123`) instead of parsing the command-line `-e`.
```

Comment:
```suggestion
If you don't call `Target.initEnvironment()` before you use the `Environment` module, 
then FAKE will only use system and inline environment variables 
(e.g. Windows: `set mything=123`) instead of parsing the command line `-e` params.
```